web/vite.config.cts

```diff
@@ -20,6 +20,24 @@ const manualChunkMap = {
   'excludedAggregatedExchanges.json': 'config',
 };
 
+const sentryPluginOptions: SentryVitePluginOptions = {
+  org: 'electricitymaps',
+  project: 'app-web',
+
+  // Auth tokens can be obtained from https://sentry.io/settings/account/api/auth-tokens/
+  // and needs the `project:releases` and `org:read` scopes
+  authToken: process.env.SENTRY_AUTH_TOKEN,
+  sourcemaps: {
+    // Specify the directory containing build artifacts
+    assets: ['./dist'],
+  },
+
+  release: {
+    // Optionally uncomment the line below to override automatic release name detection
+    name: process.env.npm_package_version,
+  },
+};
```

Comment:
Could you just double check this @tonypls before we proceed?

I bumped the plugin to the latest version (so we are up to date).